index.js

```diff
@@ -0,0 +1,10 @@
+const S = require('./dist/swipl/swipl-web');
+
+S().then(x => {
```

Comment:
This code seems to be unused and leftover from testing?

Reply:
That file is from the first commit of this PR - and has already been removed. [This is the diff I am seeing for the whole PR](https://github.com/rla/npm-swipl-wasm/pull/8/files)